src/ua.c

```diff
@@ -1013,13 +1055,14 @@ int ua_connect_dir(struct ua *ua, struct call **callp,
 	if (err)
 		goto out;
 
+	pl.p = (char *)dialbuf->buf;
+	pl.l = dialbuf->end;
+
+	(void)ua_check_dst_ip(ua, &pl);
```

Comment:
what is the purpose of checking the dst IP here ?

if you only want to check IP-address, then I think explicit logic is better:

```c
if (ip_address)
    ua_check_dst_ip();
```

when reading this code, it is not really possible to know what the function is doing
with checking the implementation

Reply:
- I remove the function and put the body into `ua_connect_dir()`.
- Another question is, why we should store the destination address in `ua`? The only reason is that we don't have the request URI in `ua_call_alloc()`. And we don't want to change the API. Right?